server/routes/appointments/appointmentsController.test.ts

```diff
@@ -610,13 +614,15 @@ describe('Scheduling a delivery session', () => {
         interventionsService.getSentReferral.mockResolvedValue(sentReferralFactory.build())
         interventionsService.getIntervention.mockResolvedValue(interventionFactory.build())
 
+        const today = new Date()
+
         await request(app)
           .post(`/service-provider/action-plan/${actionPlan.id}/sessions/1/edit/${draftBooking.id}/details`)
           .type('form')
           .send({
-            'date-day': '24',
-            'date-month': '3',
-            'date-year': '2025',
+            'date-year': today.getFullYear(),
+            'date-month': today.getUTCMonth() + 1,
+            'date-day': today.getUTCDate(),
             'time-hour': '9',
```

Comment:
I think these tests are going to start failing as soon as the clocks change - the test is now sending in the current date instead of a fixed date, and the following `expect(draftsService.updateDraft)` checks for a time of `T08:02`

Reply:
Thanks, had convo with David on Friday about this too - he suggested we should be using static data for testing to keep things predictable and simple for us. I've made that change which will solve this problem too